orb/core/column.py

```diff
@@ -419,7 +419,7 @@ def restore(self, value, context=None, inflated=True):
             if not isinstance(value, dict):
                 default_locale = locales[0]
                 if default_locale == 'all':
-                    default_locale = orb.system.settings().default_locale
```

Comment:
`settings()` doesn't need to be a method.  only used here and in `pyramid_orb`, is a minimal impact change